components/aws/sagemaker/tests/integration_tests/utils/__init__.py

```diff
@@ -8,6 +8,7 @@
 import shutil
 
 from sagemaker.amazon.amazon_estimator import get_image_uri
```

Comment:
deprecated https://github.com/aws/sagemaker-python-sdk/blame/master/CHANGELOG.md#L38

use `image_uris.retrieve()`

Reply:
That deprecated version is for SageMaker SDK 2.0 @akartsky

Reply:
This is the only place we are using SageMaker SDK in components code 
if we gonna add a new dependency might as well use the latest version

Reply:
you are right we can choose to keep using the old version 

for some reason i thought this was added to _utils.py file as a new dependency so i thought of using the latest one https://github.com/kubeflow/pipelines/blob/master/components/aws/sagemaker/common/_utils.py#L39